pxr/usd/usdLux/lightAPI.h

```diff
@@ -62,6 +62,38 @@ class SdfAssetPath;
 /// regardless of whether LightAPI is included as a built-in API of the prim 
 /// type (e.g. RectLight or DistantLight) or is applied directly to a Gprim 
 /// that should be treated as a light.
+///
+/// <b>Quantities and Units</b>
```

Comment:
I could be wrong about this, but for the usdLux header comment changes, I believe you might need to update these first in usdLux's schema.usda and then use usdGenSchema.py to generate the headers with the updated doxygen comments. See for example https://github.com/PixarAnimationStudios/OpenUSD/blob/dev/pxr/usd/usdLux/schema.usda#L49-L54